_pages/how-we-work/tools/murally.md

```diff
@@ -15,3 +15,5 @@ Because Mural is a web application, there’s no installation necessary. You wil
 - Mural consists of “rooms” (for example, FEC) and “murals” (for example, User Research). There’s only one “team:” 18F.
 - Rooms can be public or private to the rest of the team. If rooms within the 18F group are set to public, anyone at 18F can view them. If they’re private, only people specifically invited to them can see them.
 - You can give access to individual boards or rooms to anyone, including people outside of 18F or GSA, and doing so will not give them access to other public rooms without an administrator’s approval.
```

Comment:
Can you clarify here the process for giving someone access and what they'll be able to do - will they be able to edit, comment, participate in polls?

Reply:
@hannahkane can you check out the changes I made and see what you think? Thank you!